src/components/FormikAutocomplete.jsx

```diff
@@ -23,10 +25,23 @@ function FormikAutocomplete(props) {
         id={id}
         multiple
         value={values}
-        options={options}
+        inputValue={inputValue}
         onChange={(e, val) =>
-          id === 'tags' ? formatTags(val) : formik.setFieldValue(id, val)
+          onChange ? onChange(val) : formik.setFieldValue(id, val)
         }
+        onInputChange={(event, newInputValue) => {
+          const options = newInputValue.split(',');
+
+          if (options.length > 1) {
+            const vals = values.concat(
+              options.map((x) => x.trim()).filter((x) => x)
+            );
+            setInputValue('');
```

Comment:
nit: keep the last value as the input value? Otherwise I can imagine copying and pasting something like "Foo, Bar, and Baz" and `and Baz` become its own value.

Reply:
im not understanding

Reply:
Whatever the last value parsed is. Looks like it's a one-line change.
```suggestion
            const vals = values.concat(
              options.map((x) => x.trim()).filter((x) => x)
            );
            setInputValue(vals.pop());
```

Reply:
the problem with this is that it doesn't work anymore if you are typing in the tags. You would have to enter the comma in between text that is already there. I'm going to omit this since I don't think many users will copy/paste tags 🤔